tests/src/Kernel/TranslationRequestLogEntityTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation\Kernel;
+
+use Drupal\oe_translation\Entity\TranslationRequestLog;
+
+/**
+ * Tests the 'Translation request log' entity.
+ */
+class TranslationRequestLogEntityTest extends TranslationKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installEntitySchema('oe_translation_request_log');
+  }
+
+  /**
+   * Tests Translation Request entities.
+   */
+  public function testTranslationRequestLogEntity(): void {
+    // Create a translation request log entity.
+    $translation_request_log_storage = $this->container->get('entity_type.manager')->getStorage('oe_translation_request_log');
+    /** @var \Drupal\oe_translation\Entity\TranslationRequestLogInterface $translation_request_log */
+    $translation_request_log = $translation_request_log_storage->create([
+      'message' => '@status: The translation request message.',
+      'variables' => [
+        '@status' => 'Draft',
+      ],
+    ]);
+    $translation_request_log->save();
```

Comment:
No need for this save, you are testing the methods on the entity only. So no storage needed